src/openrct2/peep/Peep.cpp

```diff
@@ -405,7 +405,7 @@ std::optional<CoordsXY> Peep::UpdateAction(int16_t& xy_distance)
 
     xy_distance = x_delta + y_delta;
 
-    if (Action == PeepActionType::None1 || Action == PeepActionType::None2)
+    if (IsWalking())
```

Comment:
This used to consider both, so shouldn't it be:
```suggestion
    if (IsActionInterruptable())
```
?

Reply:
Its an redundant condition, few lines above it will always convert the action from Idle into Walking, having Idle is impossible in that statement.